### PR TITLE
Vertex Synapse Integration

### PR DESCRIPTION
**Pull request by cmd-not-found**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

- [x] Done

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Community contribution of the Synapse integration by the Vertex Project. 

> *Synapse is a “batteries included” intelligence analysis platform designed to help analysts and algorithms answer complex questions which require the fusion of data across different disciplines.*

- The project and code base are all open source and available here: [ref](https://github.com/vertexproject/synapse)
- The docs are available here: [ref](https://synapse.docs.vertex.link/en/latest/synapse/intro.html)
- Simple quickstart guide here: [ref](https://gist.github.com/cmd-not-found/a684b7b3e6ab99de8b1491a90e101ed8)

Though Synapse is extensible to many different disciplines with its underlying datamodel, it's rooted in threat intelligence analysis and can be most easily thought of as a "Threat Intelligence Platform". 

> **NOTE:** I have no affiliation with the team @ The Vertex Project. This is a community contribution. 

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Synapse/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Synapse",
+    "description": "Vertex Synapse intelligence analysis framework.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "cmd-not-found",
```
Comment (orlichter1) [inline Packs/Synapse/pack_metadata.json:6]: Could you please fill your name instead of your GitHub user?

**Review by orlichter1 — COMMENTED** [review]
Please replace the logo with the following logo
vertex synapse

**Review by orlichter1 — APPROVED** [review]

File: Packs/Synapse/Integrations/Synapse/README.md [hunk 2]
```diff
@@ -0,0 +1,736 @@
+Synapse intelligence analysis platform.
+This integration was integrated and tested with version xx of Synapse
```
Comment (orlichter1) [inline Packs/Synapse/Integrations/Synapse/README.md:2]: Last thing - could you please update what version of Synapse you worked on?

Comment (cmd-not-found) [reply Packs/Synapse/Integrations/Synapse/README.md]: @orlichter1  - updated version in `README.md` as requested and also bumped to latest version of docker image `demisto/aiohttp:1.0.0.12410`.

**Pull request merged**